inference-engine/tests/functional/plugin/cpu/single_layer_tests/depth_to_space.cpp

```diff
@@ -161,35 +200,145 @@ const std::vector<CPUSpecificParams> CPUParamsBS2_5D = {
+                         testing::Combine(
+                                 testing::ValuesIn(static_shapes_to_test_representation(inputShapesBS2_5D)),
+                                 testing::ValuesIn(inputElementType),
+                                 testing::ValuesIn(depthToSpaceModes),
+                                 testing::Values(1, 2),
+                                 testing::ValuesIn(filterCPUInfoForDevice(CPUParamsBS2_5D))),
+                         DepthToSpaceLayerCPUTest::getTestCaseName);
 
 const std::vector<CPUSpecificParams> CPUParamsBS3_5D = {
         cpuParams_ndhwc_avx2,
         cpuParams_ndhwc_sse42,
         cpuParams_ndhwc_ref,
 };
 
-const auto depthToSpaceBS3_5DParams = testing::Combine(
-        testing::Combine(
-                testing::ValuesIn(inputShapesBS3_5D),
-                testing::ValuesIn(inputPrecisions),
-                testing::ValuesIn(depthToSpaceModes),
-                testing::Values(1, 3),
-                testing::Values(CommonTestUtils::DEVICE_CPU)),
-        ::testing::ValuesIn(filterCPUInfoForDevice(CPUParamsBS3_5D))
-);
+INSTANTIATE_TEST_SUITE_P(smoke_CPUDepthToSpaceStaticBS3_5D, DepthToSpaceLayerCPUTest,
+                         testing::Combine(
+                                 testing::ValuesIn(static_shapes_to_test_representation(inputShapesBS3_5D)),
+                                 testing::ValuesIn(inputElementType),
+                                 testing::ValuesIn(depthToSpaceModes),
+                                 testing::Values(1, 3),
+                                 testing::ValuesIn(filterCPUInfoForDevice(CPUParamsBS3_5D))),
+                         DepthToSpaceLayerCPUTest::getTestCaseName);
+
+} // namespace static_shapes
+/* *========================* *==================* *========================* */
+
+
+/* *========================* Dynamic Shapes Tests *========================* */
+namespace dynamic_shapes {
+
+const auto cpuParams_avx512 = CPUSpecificParams {{}, {}, {"jit_avx512"}, {"jit_avx512"}};
+const auto cpuParams_avx512_nChw16c = CPUSpecificParams {{nChw16c}, {nChw16c}, {"jit_avx512"}, {"jit_avx512"}};
+const auto cpuParams_avx512_nCdhw16c = CPUSpecificParams {{nCdhw16c}, {nCdhw16c}, {"jit_avx512"}, {"jit_avx512"}};
+
+const auto cpuParams_avx2 = CPUSpecificParams {{}, {}, {"jit_avx2"}, {"jit_avx2"}};
+const auto cpuParams_avx2_nChw8c = CPUSpecificParams {{nChw8c}, {nChw8c}, {"jit_avx2"}, {"jit_avx2"}};
+const auto cpuParams_avx2_nCdhw8c = CPUSpecificParams {{nCdhw8c}, {nCdhw8c}, {"jit_avx2"}, {"jit_avx2"}};
+
+const auto cpuParams_sse42 = CPUSpecificParams {{}, {}, {"jit_sse42"}, {"jit_sse42"}};
+const auto cpuParams_sse42_nhwc = CPUSpecificParams {{nhwc}, {nhwc}, {"jit_sse42"}, {"jit_sse42"}};
+const auto cpuParams_sse42_ndhwc = CPUSpecificParams {{ndhwc}, {ndhwc}, {"jit_sse42"}, {"jit_sse42"}};
+
+const auto cpuParams_ref = CPUSpecificParams {{}, {}, {"ref_any"}, {"ref_any"}};
```

Comment:
Looks like a copy of the previous declaration. Is not it better to store them in one place? The same is true for its vectors. BTW this parameters does not work in regard to impl type, since it is chosen considering the highest available ISA on the particular machine, and, it is not actually checked in the tests, since the expected impl_type is ignored and set by  `getPrimitiveType()` call.